internal/service/ec2/ec2_instance_migrate_test.go

```diff
@@ -166,3 +168,142 @@ func TestInstanceMigrateState_empty(t *testing.T) {
 		t.Fatalf("err: %#v", err)
 	}
 }
+
+func TestInstanceStateUpgradeV1(t *testing.T) {
```

Comment:
In addition to unit testing, could we add one or two variations of acceptance tests which upgrade impacted configurations?

Reply:
Added